tests/playTests.sh

```diff
@@ -186,7 +186,7 @@ fi
 
 println "\n===>  simple tests "
 
-datagen > tmp
+datagen -g500K > tmp
```

Comment:
any specific reason for this change ?

Reply:
Yes.
One of the following tests checks that we fail with "-M2K". This test started failing (i.e decompression succeeded) when I introduced the changes in the branch.
Reason being that I increased the CLI's buffer sizes (and changed buffer usage pattern) resulting in less mallocs during decompression. Increasing the data size forces the lib to allocate space for the window. (**note this is as far as I currently understand the code, might have missed something in this explanation).

Reply:
OK, 
so maybe it is this one `-M2K` test (or series of tests) that needs to generate and use its own specific source,
rather than impacting a generic sample used by many other tests.

Reply:
Done